pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -98,21 +108,53 @@ var _ = Describe("Pod Network", func() {
 
 		masqueradeGwStr = "10.0.2.1/30"
 		masqueradeGwAddr, _ = netlink.ParseAddr(masqueradeGwStr)
+		masqueradeGwIp = masqueradeGwAddr.IP.String()
 		masqueradeVmStr = "10.0.2.2/30"
 		masqueradeVmAddr, _ = netlink.ParseAddr(masqueradeVmStr)
+		masqueradeVmIp = masqueradeVmAddr.IP.String()
+		masqueradeIpv6GwStr = "fd2e:f1fe:9490:a8ff::1/120"
+		masqueradeIpv6GwAddr, _ = netlink.ParseAddr(masqueradeIpv6GwStr)
+		masqueradeGwIpv6 = masqueradeIpv6GwAddr.IP.String()
+		masqueradeIpv6VmStr = "fd2e:f1fe:9490:a8ff::2/120"
+		masqueradeIpv6VmAddr, _ = netlink.ParseAddr(masqueradeIpv6VmStr)
+		masqueradeVmIpv6 = masqueradeIpv6VmAddr.IP.String()
 		masqueradeDummyName = fmt.Sprintf("%s-nic", api.DefaultBridgeName)
 		masqueradeDummy = &netlink.Dummy{LinkAttrs: netlink.LinkAttrs{Name: masqueradeDummyName}}
 		masqueradeTestNic = &VIF{Name: podInterface,
-			IP:      *masqueradeVmAddr,
-			MAC:     fakeMac,
-			Mtu:     1410,
-			Gateway: masqueradeGwAddr.IP.To4()}
+			IP:          *masqueradeVmAddr,
+			IPv6:        *masqueradeIpv6VmAddr,
+			MAC:         fakeMac,
+			Mtu:         1410,
+			Gateway:     masqueradeGwAddr.IP.To4(),
+			GatewayIpv6: masqueradeIpv6GwAddr.IP.To16()}
```

Comment:
We are targeting ipv6 only right now. Is this separation already done to have a way in the future to express dual-stack requirements?

Reply:
yes